### ISSUE TITLE
Log Error in 2023.2

### ISSUE THREAD

**Issue by maegibbons** [gndean/home-assistant-hypervolt-charger#17]
You have probably seen this but thought I would highlight just in case:

```
Logger: homeassistant.components.sensor
Source: components/sensor/__init__.py:503
Integration: Sensor (documentation, issues)
First occurred: 05:41:44 (1 occurrences)
Last logged: 05:41:44

Entity sensor.hypervolt_session_carbon_saved (<class 'custom_components.hypervolt_charger.sensor.HypervoltSensor'>) is using state class 'measurement' which is impossible considering device class ('weight') it is using; expected None or one of 'total'; Please update your configuration if your entity is manually configured, otherwise report it to the custom integration author.
```

**Comment by gndean** [comment]
Thanks for the info. This looks like it might be an issue with Home Assistant core: https://github.com/home-assistant/core/issues/86949
I'm not really sure this sensor is wrong 🤷‍♂️

Let's see if it's fixed in a future version of HA. In any case, I don't think it was have any practical effect.

**Comment by gndean** [comment]
I'm no longer seeing this with 2023.3 so it looks like it was indeed a bug in HA and now fixed.

@maegibbons, can you confirm the same?

**Comment by maegibbons** [comment]
Yes not seeing this error any more.  

So looks fixed in 2023.3
